<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWaypointEdit</class>
 <widget class="QDialog" name="DialogWaypointEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Waypoint</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
}

#frame, #frame_2, #frame_3
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}


QLabel
{
	color: rgb(107, 239, 0);
}

#listRSBN
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);

	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

#listRSBN::item:selected
{
	background-color: rgb(107, 239, 0);
	color: rgb(0, 30, 0);
}

QPushButton
{
	background-color: rgb(73, 163, 0);
	color: rgb(0, 0, 0);
}

QComboBox
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
	selection-background-color: rgb(107, 239, 0);
    selection-color: rgb(0, 0, 0);
}

QComboBox QAbstractItemView
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QLineEdit
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QSpinBox
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QDoubleSpinBox
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QCheckBox
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
}

QCheckBox::indicator::checked
{
	background-color: rgb(107, 239, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QCheckBox::indicator::unchecked
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
	border: 1px solid;
	border-color: rgb(73, 163, 0);
}

QMenu
{
	background-color: rgb(0, 30, 0);
	color: rgb(107, 239, 0);
}

QMenu::item:selected
{
	background-color: rgb(107, 239, 0);
	color: rgb(0, 0, 0);
}

QMenu::item:disabled
{
	background-color: rgb(0, 15, 0);
	color: rgb(27, 159, 0);;
}

</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <widget class="QLineEdit" name="lineEdit_Name">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>301</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Identifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Identifier">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Country (ISO)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Country">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>611</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <widget class="QLabel" name="label_Freq_TransAlt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Frequency</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RangeTransLevel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Range (NM)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_AngleDev_LongestRwy">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Angle Deviation</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Range">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_AngleDev">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Freq">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>611</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Latitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>161</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Longitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Elevation (ft)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Lat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>13</number>
    </property>
    <property name="minimum">
     <double>-90.000000000000000</double>
    </property>
    <property name="maximum">
     <double>90.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Lon">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>201</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>13</number>
    </property>
    <property name="minimum">
     <double>-180.000000000000000</double>
    </property>
    <property name="maximum">
     <double>180.000000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Elev">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_Alt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Altitude (ft)</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Position</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>480</y>
     <width>140</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SaveCurrent">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>480</y>
     <width>140</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Save in flightplan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CreateNew">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>240</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Save and add to XNVU library</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Identifier</tabstop>
  <tabstop>lineEdit_Name</tabstop>
  <tabstop>lineEdit_Country</tabstop>
  <tabstop>doubleSpinBox_Lat</tabstop>
  <tabstop>doubleSpinBox_Lon</tabstop>
  <tabstop>spinBox_Elev</tabstop>
  <tabstop>spinBox_Alt</tabstop>
  <tabstop>comboBox_Type</tabstop>
  <tabstop>doubleSpinBox_Freq</tabstop>
  <tabstop>spinBox_Range</tabstop>
  <tabstop>doubleSpinBox_AngleDev</tabstop>
  <tabstop>pushButton_SaveCurrent</tabstop>
  <tabstop>pushButton_CreateNew</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
